zipkin-ui/test/component_data/default.test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  parseApiQuery
+} from '../../js/component_data/default';
+
+chai.config.truncateThreshold = 0;
+
+describe('parseApiQuery', () => {
+  const should = require('chai').should();
+
+  it('should not require startTs', () => {
+    const parsed = parseApiQuery('?endTs=1459169770000');
+
+    parsed.endTs.should.equal('1459169770000');
+    should.equal(parsed.startTs, undefined);
+    should.equal(parsed.lookback, undefined);
+  });
+
+  it('should replace startTs with lookback', () => {
+    const parsed = parseApiQuery('?startTs=1459169760000&endTs=1459169770000');
+
+    parsed.endTs.should.equal('1459169770000');
+    parsed.lookback.should.equal(10000);
```

Comment:
Isn't it a little bit inconsistent that endTs is a string, while lookback is a number?